### DISCUSSION TITLE
Linting commit messages with Github Actions

### DISCUSSION THREAD

**Discussion by GeorgeLeithead** [CommunityToolkit/Maui#586] [General]
I read this article from "Dev Community", and thought it 'might' be good to add.

[Linting commit messages with Github Actions](https://dev.to/brianvu/linting-commit-messages-with-github-actions-388c)

**Comment by pictos** [comment]
I don't like this idea too much... Our idea is to be as simple as possible for new devs to start their journey in the OSS. Adding a lot of rules will make that difficult. IMHO, we're good in terms of code style, commit, etc. TBH in terms of code style, we have a bot that adjusts what we have missed during the code review phase.

**Reply by GeorgeLeithead** [reply]
I'm raising this for discussion, and I think your point is certainly valid. One benefit I can see is guiding contributors on potential best practice.

**Reply by pictos** [reply]
Don't get me wrong. I love that you're bringing new ideas and engaging discussions! I just didn't like this particular one. Don't let me or anyone else intimidate you, if you think it's important raise that up. I'll pay more attention in my tone (; , english isn't my primary laguage so sometimes I can express a wrong feeling

**Reply by GeorgeLeithead** [reply]
Oh no @Pictos!  Your tone is perfectly fine.  I admire people who are bi-lingual.
This is a discussion and all opinions and arguments deserve a voice.  I appreciate the involvement, community feedback and positive feedback.

**Comment by VladislavAntonyuk** [comment]
We merge and squash commits, so it is up to developer how to name commits.

**Comment by CliffAgius** [comment]
Closed as answered
